src/Resources/Locales/en.json

```diff
@@ -1358,7 +1360,10 @@
         "noADTInstanceUrlTitle": "ADT instance not set",
         "noADTInstanceUrlMessage": "Your ADT instance url is not set. Click below to set it.",
         "configureEnvironment": "Configure environment",
+        "networkErrorTitle": "Connection error",
         "networkErrorMessage": "This could be a CORS issue, invalid url or a network error. Please contact the owner of the resource or try again later.",
+        "applyUpdates": "Apply updates",
+        "setCorsPropertiesNotAuthorizedMessage": "You do not have access to the storage account to make this change. Please contact the resource owner to make the required changes on CORS settings of the storage account.",
```

Comment:
do we still direct them to the docs somehwere so they know _which_ CORS settings to fix?

Reply:
So I am going to ask Bailey (our content person) to add these headers to the [prerequisite docs](https://learn.microsoft.com/en-us/azure/digital-twins/how-to-use-3d-scenes-studio#prerequisites) where we talk about CORS headers to add. But should we update these docs now @darsney ?